ndk/src/event.rs

```diff
@@ -656,6 +670,14 @@ impl<'a> Pointer<'a> {
     pub fn touch_minor(&self) -> f32 {
         unsafe { ffi::AMotionEvent_getTouchMinor(self.event.as_ptr(), self.index as ffi::size_t) }
     }
+
+    #[inline]
+    pub fn tool_type(&self) -> ToolType {
+        let tool_type = unsafe {
+            ffi::AMotionEvent_getToolType(self.event.as_ptr(), self.index as ffi::size_t) as u32
+        };
+        tool_type.try_into().unwrap()
```

Comment:
It seems that in some cases we use `.try_into().unwrap_or(XXX::Unknown)`, but I'd have to ask @mb64 why that wasn't done in all cases.

Though I think I'd rather panic than consider every yet-unknown value to be equivalent to the literal `_UNKNOWN` constant, as this typically means the NDK or these enums need an update (which I'd like to mitigate even further by generating these types and derives in bindgen directly, but that hasn't materialized yet and especially the event code uses anonymous enums and typedefs which can't yet be interpreted that way). For example, `Source` is missing `Hdmi` and `Sensor` :)